### DISCUSSION TITLE
Reusable content blocks

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#19] [Feature Requests]
### What's your feature/idea?

The option to configure an editor or a group of editors and save this as a "reusable"/global/template block which can then be used again and again.
Making a change to the "reusable"/global/template block would then be done from one place, and update all the places where the block is in use.

### What's your desired outcome?

I might have explained that above already ☝️ 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

I've seen this as a feature request earlier previously, so forgive me, if this is something you're already working on! 😄

**Comment by taszap** [comment]
I will close this in favour of #121 as they are related
